cluster_pack/filesystem.py

```diff
@@ -41,56 +45,44 @@ def _expose_methods(child_class: Any, base_class: Any, ignored: List[str] = [])
     :param child_class: instance of child class to add the methods to
     :param ignored: methods that will be redefined manually
     """
-    method_list = [func for func in dir(base_class)
-                   if callable(getattr(base_class, func))
-                   and not func.startswith("__")
-                   and not [f for f in ignored if func.startswith(f)]]
+    method_list = [

```

Comment:
They are many (mainly actually) lines in this PR where you don't actually change things but it is just formatting, this makes the PR very hard ot review because real changes re completely hidden into this. Can you remove all the formatting changes and upload something cleaner pleae ?